roles/storage-cns/tasks/deprovision.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Login As Super User
+  command: "oc login -u {{ admin_user }} -p {{ admin_password }}"
+  when: cluster=="openshift"
+        and admin_user is defined
+        and admin_password is defined
+
+- name: Render storage-cns deployment yaml
+  template:
+    src: "{{ storage_cns_template_dir }}/storage-cns.yml"
```

Comment:
Above it says ```There is no need to change this value```. By default, template will look in the ```templates``` directory so you drop this variable and change to ```src: storage-cns.yml```.

Reply:
Thanks.  Fixed.